precode.go

```diff
@@ -39,14 +41,73 @@ var tasks = map[string]Task{
 	},
 }
 
-// Ниже напишите обработчики для каждого эндпоинта
-// ...
+// Обработчик выдающий все задачи
+func getTask(w http.ResponseWriter, r *http.Request) {
+	resp, err := json.Marshal(tasks)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest) //Ошибка 400 bad request
+		return                                            //завершаем в случае ошибки
+	}
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK) //Статус 200 ОК
+	w.Write(resp)
+}
 
-func main() {
-	r := chi.NewRouter()
+// Обработчик выдающий задачу по ID
+func getTaskID(w http.ResponseWriter, r *http.Request) {
+	id := chi.URLParam(r, "id")
+	task, ok := tasks[id]
+	if !ok {
+		http.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest) //400 bad request
```

Comment:
Можно что-то свое написать
```suggestion
		http.Error(w, "task not found", http.StatusBadRequest) //400 bad request
```